assets/src/scripts/components/graph/state/layout.js

```diff
@@ -174,16 +181,24 @@ export const getChartPosition = memoize((opts, chartType) => createSelector(
                 };
             case 'lithology':
                 return {
-                    x: 0,
+                    // x: adjusts the horizontal starting point of well lithology chart within the SVG view box
+                    // Note: (x: viewBox.right * 1) well construction chart is out of the SVG view box to the right
+                    // Note: (x: viewBox.right * 0) start of well construction chart touches edge of SVG view box to the left
+                    x: viewBox.right * 0.13,
```

Comment:
*blush* I actually couldn't get the code to work that would give me access to the width of either of the well diagram axes. I made several attempts without success. This may be an area where a more experienced developer would have more insight. The hard coded numbers do work for the vast majority of situations, and digging deeper didn't seem to reap much reward.